paasta_tools/firewall_update.py

```diff
@@ -12,6 +12,7 @@
 from inotify.adapters import Inotify
```

Comment:
Yeah agreed. :/ Will think about this some more. Maybe we just don't do it for the time being.